cypress/e2e/1-testing/login-fail.cy.js

```diff
@@ -0,0 +1,19 @@
+describe("Login Fail", () => {
+  it("should show error message when logging in with invalid credentials", () => {
+    cy.visit("https://murots.github.io/Workflow_ca/");
```

Comment:
```suggestion
    cy.visit("/");
```

When you test a remote deployment, instead of the local code - you are not providing any utility at all. Tests should be run on "hot" code in your branch, rather than "cold" code that has already been deployed.

This is because the decision "should I deploy this" should only be "yes" if the work has been tested beforehand, not afterwards.

Cypress runs its own webserver, do you can just hit "/" instead.

Reply:
Thank you. I have replaced the remote link with "/". When I run everything in cypress, all tests passes. But they now all fail in my automated e2e-testing, and I cant understand why. Can you please point me in the right direction?

Reply:
I went back to remote links just to check if this was the issue, and now all the tests passes. Not sure why?

Reply:
Yeah, it's the same issue with the e2e tests that I thought I mentioned elsewhere:

`cy.get('#registerModal').should('not.be.visible');`

If you remove this, your test succeeds and will do on actions too.

Reply:
Ah, I must have missed that. Thank you. All passes now.